# if in jail
bool jailed

# where to start push from
geometry_msgs/Pose start
# where to stop pushing
geometry_msgs/Pose goal
# the target to be pushed
Target target

